### ISSUE TITLE
Add Project Reference on OSX fails

### ISSUE THREAD

**Issue by thomaschrstnsn** [ionide/ionide-vscode-fsharp#118]
Using "Add Project Reference" fails on OSX (homebrew mono)

The Forge log contains the following:

```
forge add project -n /Users/thomas/src/fsdemo/src/MyLibrary/MyLibrary.fsproj -p /Users/thomas/src/fsdemo/src/console/console.fsproj --no-prompt
﻿﻿Unrecognized command or missing required parameter
```

The binding to Forge seems broken.

Using:
 - vscode 1.3.1 
 - Ionide-fsharp 2.2.8

**Comment by thomaschrstnsn** [comment]
On the other hand it could be same as this: https://github.com/fsprojects/Forge/issues/119

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `2.2.10`
